### DISCUSSION TITLE
Docker configuration question

### DISCUSSION THREAD

**Discussion by vyau** [milvus-io/milvus#26812] [Q&A and General discussion] [answered]
Hello:
I am running Milvus under Docker.

For my docker-compose.yaml, I am unable map two different volumes paths together.
I want to add another line to map my milvus.yaml.  Is it possible to configure volumes section to look like the following.
I am unable to bring up Milvus when I have more than one line under "volumes".

   volumes:
      - ${DOCKER_VOLUME_DIRECTORY:-.}/volumes/milvus:/var/lib/milvus
      - /local/path/to/your/milvus.yaml:/milvus/configs/milvus.yaml

**Comment by yhmo** [comment, marked as answer]
The two lines are mapping to different paths, it should be ok.

/var/lib/milvus  is the path the milvus stores its write-ahead-log files.
/milvus/configs/milvus.yaml is the configuration file. It should work well if you didn't make any changes to the default configuration file.

If it failed to start, you can use `docker logs [container id]` to export the log.
